go.mod

```diff
@@ -109,6 +110,8 @@ require (
 	sigs.k8s.io/kustomize/kustomize/v5 v5.3.0
 )
 
+require github.com/google/go-github/v55 v55.0.0 // indirect
```

Comment:
move to the indirect require block around line 155

Reply:
Alright